src/entry-client.js

```diff
@@ -67,6 +67,28 @@ const perfEnabled = () => (
 const getCurrentPerfMark = () => window.performance.getEntriesByType('mark')
     .find(m => m.name.startsWith(PERF_PREFIX) && m.name.endsWith('start'));
 
+function perfMarkNameFromRoute(route) {
+    let result;
+    const routeName = get(route, 'name') || '';
+
+    if (routeName.toLowerCase() === 'catch-all') {
```

Comment:
Hm, this concept might not work after all.  This is a bit specific to SlipStream so I don't think we want this in `vue-ssr-build`, but beyond that I don't think we would even know at the start of the route what the page type would be.  We don't know what type of page it is until after `fetchData`

So currently, setting this based on `catch-all` ends up incorrectly labelling landing pages as category:

<img width="865" alt="Screen Shot 2021-01-22 at 10 37 11 AM" src="https://user-images.githubusercontent.com/1609022/105511845-5ecccb80-5c9e-11eb-9c75-ddabbedb88fe.png">

And for all of the marks _before_ fetchData we have no way to know what sub-type it is.  I think we may need to leave this as `catch-all` unfortunately :(